src/GraphQL.Tests/Bugs/Bug1831.cs

```diff
@@ -35,8 +35,14 @@ public void TestVariableObject_InvalidType(string param)
                 Code = "KNOWN_TYPE_NAMES"
             };
             error2.AddLocation(1, 13);
-            var expected = CreateQueryResult(null, new ExecutionErrors { error1, error2 }, executed: false);
-            AssertQueryIgnoreErrors("query($arg: abcdefg) { test1 (arg: $arg) }", expected, inputs: $"{{ \"arg\": {param} }}".ToInputs(), expectedErrorCount: 2, renderErrors: true);
+            var error3 = new ValidationError(null, "5.6.1",
+               "Variable \u0027$arg\u0027 is invalid. Variable has unknown type \u0027abcdefg\u0027")
```

Comment:
@joemcbride @Shane32 Compare this message with one from `KnownTypeNamesError.UnknownTypeMessage`. They are similar in meaning, but still different. One speaks about an unknown type as such, and the second about an incorrectly set variable of an unknown type. The context is different. I think this is more an advantage than a disadvantage. You will find similar new validation messages in many more tests.